tests/action_tests.py

```diff
@@ -57,14 +64,20 @@ def test_successful_provision_account(get_patch):
 
     res_patch = mock.Mock(spec=requests.Response)
     res_patch.status_code = 200
-    res_patch.content = json.dumps(fake_content)
+    res_patch.content = json.dumps(fake_post_content)
+    post_patch.return_value = res_patch
+
+    res_patch = mock.Mock(spec=requests.Response)
+    res_patch.status_code = 200
+    res_patch.content = json.dumps(fake_get_content)
     get_patch.return_value = res_patch
+
     boompy.set_auth("account_id", "username", "password")
     boompy.actions.provisionPartnerCustomerAccount(fake_data)
```

Comment:
I know this is an old test, but just wondering - No assertions about return value or any side effects?